less/bookmarksToolbar.less

```diff
@@ -12,7 +12,7 @@
   display: flex;
   flex: 1;
   padding: 3px 10px;
-  height: 23px;
+  height: 30px;
```

Comment:
I know we want to be competitive on the amount of height our browser chrome takes up (it matters a lot to some people).  Increasing by 7px always wouldn't fly.  

I see 2 options:
i) Add a className in React code that's handled here that specifies whether or not to use 23px for no favicon or 30px for favicons. (call the class something like `showFavicons`).
or ii) Always use 23px and just resize the favicons smaller.
CC @bradleyrichter